src/components/Welcome/Welcome.js

```diff
@@ -1,56 +1,60 @@
 import React from 'react';
 import RecipeCard from '../RecipeCard/RecipeCard.js';
 import './Welcome.css';
-import { getRandomRecipe } from './../../APICalls'
+import { getRandomRecipe, getLatestRecipes } from './../../APICalls'
+import LoginButton from '../LoginButton/LoginButton';
+
 
 const Welcome = () => {
   const  userData = {}
   
   const [randomDrink, setRandomDrink] = React.useState([])
+  const [drinks, setDrinks] = React.useState([])
 
   React.useEffect(() => {
     getRandomRecipe()
     .then(data => {
       setRandomDrink(data.drinks[0])
```

Comment:
I'm not seeing any error handling path here for either of the network requests, which should be addressed if something fails server side.